.github/workflows/container-deploy.yml

```diff
@@ -0,0 +1,141 @@
+name: Container Image Deployment CI
+
+on:
+  push:
+    branches: 
+      - main
+      - deploy-*
+    tags: 
+      - v*.*.*
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    environment: ${{ startsWith(github.ref, 'refs/tags/v') && 'off-org' || 'off-net' }}
+    concurrency: ${{ startsWith(github.ref, 'refs/tags/v') && 'off-org' || 'off-net' }}
+    steps:
+    - name: Wait for container build workflow
+      uses: tomchv/wait-my-workflow@v1.1.0
+      id: wait-build
+      with:
+        token: ${{ secrets.GITHUB_TOKEN }}
+        checkName: build
+        ref: ${{ github.event.pull_request.head.sha || github.sha }}
+        intervalSeconds: 10
+        timeoutSeconds: 600 # 10m
+
+    - name: Do something if build fail
+      if: steps.wait-build.outputs.conclusion == 'failure'
+      run: echo fail && false # fail if build fail
+
+    - name: Do something if build timeout
+      if: steps.wait-build.outputs.conclusion == 'timed_out'
+      run: echo Timeout && false # fail if build time out
+
+    - name: Deploy Docker image to Proxmox VM
+      uses: appleboy/ssh-action@master
+      env:
+        DOCKER_CLIENT_TIMEOUT: 120
+        COMPOSE_HTTP_TIMEOUT: 120
+        PRODUCT_OPENER_DOMAIN: ${{ secrets.PRODUCT_OPENER_DOMAIN }}
+        ROBOTOFF_URL: ${{ secrets.ROBOTOFF_URL }}
+        GOOGLE_CLOUD_VISION_API_KEY: ${{ secrets.GOOGLE_CLOUD_VISION_API_KEY }}
+        CROWDIN_PROJECT_IDENTIFIER: ${{ secrets.CROWDIN_PROJECT_IDENTIFIER }}
+        CROWDIN_PROJECT_KEY: ${{ secrets.CROWDIN_PROJECT_KEY }}
+        CSRF_SECRET: ${{ secrets.CSRF_SECRET }}
+        GEOLITE2_PATH: ${{ secrets.GEOLITE2_PATH }}
```

Comment:
Is this GitHub secret set in the current test setup? I think there'd probably be a volume mount missing for it, in that case. It might be easier to employ https://github.com/openfoodfacts/openfoodfacts-server/blob/9af7d8767d3e2a84b055f96b9fa18dc3a45988e6/docker/docker-compose.geolite2.yml inside this YAML pipeline: https://github.com/openfoodfacts/openfoodfacts-server/blob/b0d5043c25dca8b99b197e2a54c95df96938f66b/docker/README.md#L15